appendix-basic65.tex

```diff
@@ -1940,6 +1945,11 @@ \subsection{COLOR}
                for screen characters. This speeds up screen
                handling, which could be useful when moving characters with different colours is
                not required.
+
+
+\item
+\item which is documented
+               on page \pageref{foregroundcommand}.
```

Comment:
Not sure what's up with this section. Seems loosely dangling in the middle of newhere. Was it a cut/paste misfire?

Reply:
yeah looks like it, removing it now